pkgs/development/compilers/bs-platform/default.nix

```diff
@@ -23,6 +23,6 @@ in
     platforms = platforms.all;
     # Currently there is an issue with aarch build in hydra
     # https://github.com/BuckleScript/bucklescript/issues/4091
-    badPlatforms = platforms.aarch64;
+    # badPlatforms = platforms.aarch64;
```

Comment:
aarch64 is still failing.

Reply:
what's the error message this time?

Reply:
Hey @bobzhang, logs here: https://logs.nix.ci/?key=nixos/nixpkgs.77726&attempt_id=25497bfb-33dc-4721-aae3-2c7d3c755f93

Reply:
I am a bit lost, by reading the logs, it still seems to be a dev build, since cppo is running, in release build, cppo is not needed